src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -466,15 +428,35 @@ public static boolean isCurrentUserCPMAdmin(String userId, Environment env) {
 	}
 
 	/*
-	 * get CoSign user
+	 * get CoSign user, for local development user will be passed from url parameter as
+	 * http://localhost:8080/?testUser=<uniqname> together with use.testUser.url from the
+	 * application.properties we will determine valid user. In short this block is mimicking
+	 * cosign user if CoSign is not enabled
```

Comment:
I strongly suggest renaming this to something like "getUserId" or "getUserLoginId" (or something even better).  Remote user is only one of the sources of data so it's misleading to put that in the name.